### ISSUE TITLE
Env vars should not be required [in progress]

### ISSUE THREAD

**Issue by randyoyarzabal** [binbashroot/provision_proxmox#13]
There should be an overridable ansible var for each ENV var.  There should NOT be any code calls to ENV vars, then by default, in the custom user vars, the user can then just lookup the ENV var, i.e.:

proxmox_api_host: "{{ lookup('env','PROXMOX_HOST') }}"
proxmox_api_pass: "{{ lookup('env','PROXMOX_PASSWORD') }}"
proxmox_api_user: "{{ lookup('env','PROXMOX_USER') }}"

These too:
export REDHAT_PASS='******'
export REDHAT_USER='[jdoe@duck.com](mailto:jdoe@duck.com)'

What this allows the user to do is naturally define things in there parameters yaml, and ansible-vault it in the native ansible way.  The calling command would optionally be:

ansible-playbook site.yml $@ --extra-vars='@user_vars.yaml' --ask-vault-pass
Or
ansible-playbook site.yml $@ --extra-vars='@user_vars.yaml' --vault-password-file=<password file>
and other strategies, etc. 

Basically, not requiring ENV vars.  It should be optional.

**Comment by binbashroot** [comment]
Agree and pushed changes
